frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const namespace = 'openshift-storage';
+const kind = 'storagecluster';
+const storageCluster = JSON.parse(
+  execSync(`kubectl get -o json -n ${namespace} ${kind}`).toString(),
+);
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+describe('Check add capacity functionality for ocs service', () => {
```

Comment:
Polarion test IDs for Cluster expansion tests (tier1) are available at https://docs.google.com/spreadsheets/d/1ry_WQsmVPdR6Elj-WGcNY9flIOQ7LEKDvE7HEs5S6a0/edit#gid=794221253&range=B2
Which of these tests that this PR is going to cover?

Reply:
@hnallurv I have added bare minimum tests required for expand flow , which are -
1) The newly added capacity takes into effect at the storage level. 
2) New osd pods corresponding to the additional capacity should be in running state.
3) 'ceph -s' should show HEALTH_OK
4) No ocs pods should get restarted unexpectedly
5) No OCP/OCS nodes should go to NotReady state

For rest other test cases, separate PR will be sent.